app/controllers/rocket_job_mission_control/jobs_controller.rb

```diff
@@ -182,6 +182,9 @@ def update_slice
       # Finds specific slice [Array]
       slice = @job.input.failed.skip(offset).first
 
+      # Converts \r\n line breaks to \n
+      updated_records[0] = updated_records[0].gsub(/\r\n/, "\n")
```

Comment:
Can we replace it with an iterator to change all records?
```suggestion
      updated_records.each { |record| record.gsub!(/\r\n/, "\n") }
```

Reply:
Added Correction.